### PR TITLE
Refactor instance-mute to use Composition API

### PR DESCRIPTION
**Pull request by pixeldesu**
# What

Use Composition API for `settings/instance-mute` page

# Why

Refactor to Composition API (per contribution guide)

### CODE REVIEW

File: packages/client/src/pages/settings/instance-mute.vue [hunk 1]
```diff
@@ -1,67 +1,53 @@
...
-			// Refresh filtered list to signal to the user how they've been saved
-			this.instanceMutes = mutes.join('\n');
-		},
+import { $i } from '@/account';
+import { i18n } from '@/i18n';
+
+const instanceMutes = ref('');
+const changed = ref(false);
+
+instanceMutes.value = $i!.mutedInstances.join('\n');
```
Comment (Johann150) [inline packages/client/src/pages/settings/instance-mute.vue]: I think setting up `instanceMutes` could be merged into one.
```suggestion
const instanceMutes = ref($i!.mutedInstances.join('\n'));
const changed = ref(false);
```

Comment (pixeldesu) [reply packages/client/src/pages/settings/instance-mute.vue]: Yeah, that makes a lot of sense actually!

**Review by Johann150 — APPROVED** [review]

File: packages/client/src/pages/settings/instance-mute.vue [hunk 2]
```diff
@@ -1,67 +1,51 @@
...
+async function save() {
+	let mutes = instanceMutes.value
+		.trim().split('\n')
+		.map(el => el.trim())
+		.filter(el => el);
+	
+	await os.api('i/update', {
+		mutedInstances: mutes,
+	});
+
```
Comment (Johann150) [inline packages/client/src/pages/settings/instance-mute.vue]: git complains about whitespace
```suggestion

	await os.api('i/update', {
		mutedInstances: mutes,
	});

```

**Comment by syuilo** [comment]
🙏

**Pull request merged**